crates/cli/src/main.rs

```diff
@@ -170,6 +170,14 @@ struct Parse {
     /// The path to the tree-sitter grammar directory
     #[arg(long, short = 'p')]
     pub grammar_path: Option<PathBuf>,
+    /// Don't force rebuild the parser when `--grammar_path` is specified
```

Comment:
These comments are a bit misleading, if I understand the PR correctly. Isn't it more of a
```rust
/// `--grammar_path` implies force rebuild
```
?